-
  I schedule phonecall to another phonecall, so first i create record of schedule call.
-
  !record {model: crm.phonecall2phonecall, id: crm_schedule_phonecall}:
    name: 'Test Discuss Review Process'
    categ_id: crm.categ_phone1
    date: !eval time.strftime('%Y-%m-%d') 
-
  I Convert schedule phonecall.
-
  !python {model: crm.phonecall2phonecall}: |
    self.action_apply(cr, uid, [ref('crm_schedule_phonecall')], {'active_id': ref('crm_case_phone04')})
-
  I check for another schedule call.
-
  !python {model: crm.phonecall}: |
    phonecall = self.search(cr, uid, [('name', '=', 'Test Discuss Review Process')])
    assert phonecall, "Do not create another schedule call"
    schedule_phonecall = self.browse(cr, uid, phonecall)[0]
    assert schedule_phonecall.user_id.id == ref("base.user_root"), "Responsible person not assign"
    assert schedule_phonecall.section_id.id == ref("crm.section_sales_department"), "Do not assign proper section"
    assert schedule_phonecall.categ_id.id == ref("crm.categ_phone1"), "Category of phonecall is not apply"
    assert schedule_phonecall.partner_id.id == ref("base.res_partner_tinyatwork"), "Partner missmatch"